samples/queuesGettingStarted.ts

```diff
@@ -0,0 +1,71 @@
+import { OnMessage, OnError, MessagingError, delay, ServiceBusMessage, ReceiveMode, generateUuid, Namespace } from "../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const str = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const path = process.env.QUEUE_NAME || "";
+console.log("str: ", str);
+console.log("path: ", path);
+
+let nsSend: Namespace;
```

Comment:
I think these can be moved into the actual methods that use them. Any particular reason to have them as globals here?

Reply:
Are you talking about the namespaces?

Reply:
Yes -- can those be moved inside the methods that use them?

Reply:
Yes!

Reply:
Not if you plan to close the namespace from main().

Reply:
Also, any reason to use 2 separate instances of the namespace?

Reply:
I'm closing the namespaces inside the methods(send, receive).

Reply:
No specific reason but I was following the .NET code that's why I created 2 different namespaces.